.github/workflows/update-readme.yml

```diff
@@ -21,5 +21,17 @@ jobs:
       - name: Compile parsers
         run: ./nvim.appimage --headless -c "TSInstallSync all" -c "q"
 
+      # inspired by nvim-lspconfigs
       - name: Check README
-        run: ./nvim.appimage --headless -c "luafile ./scripts/update-readme.lua" -c "q"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we need to set that

Reply:
You need to do that on Travis. But it would be strange if you need Github to authenticate it self on Github. There's also a magic environment variable to do this on Gitlab. It gives the current user the rights of the triggered. But apparently the jobs in not executed with the rights of the user but with the rights of the GH Actions bot.

Reply:
Our secrets store is empty though so this env varialbe will not be expanded to anything

Reply:
Can you remove that `GITHUB_TOKEN` ?

Reply:
GITHUB_TOKEN is necessary for pushing.